Makefile

```diff
@@ -1,5 +1,118 @@
 # Citus toplevel Makefile
```

Comment:
Why is this PR changing this?

Reply:
Windows builds don't use Make, they instead use visual studio. There's a script in the postgres repo, Mkvcbuild.pm, which will parse our Makefile and generate the correct visual studio project file. Here we're getting our Makefile into a form which it understands.

Probably a better alternative would be to switch over to CMake, but that's going to be a large project.